src/Compilers/Core/Portable/DiagnosticAnalyzer/IAnalyzerPathResolver.cs

```diff
@@ -0,0 +1,52 @@
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// This interface allows hosts to control where an analyzer is loaded from. It can redirect the path 
+    /// originally passed to the compiler to a new path. Or it can take ownership of a path to prevent 
```

Comment:
```suggestion
    /// originally passed to the compiler to a new path (called the "real path"). Or it can take ownership of a path to prevent 
```

(note: i'm only assuming i'm correct that that's what the 'real path' is.  But it would help in terms of consistent terminology throughout this space).